### DISCUSSION TITLE
Possibility for userAgentData

### DISCUSSION THREAD

**Discussion by AKaravas** [faisalman/ua-parser-js#712] [General]
Modern browsers have implemented the `navigator.userAgentData` API. Is this supported? If not, are there plans to do so? On chrome and MacOS is active, and on firefox on an experimental version. 

Best regards

**Comment by faisalman** [comment]
`navigator.userAgentData` has been supported since `v2.0`:

https://github.com/faisalman/ua-parser-js/blob/5328642e1831e4e7adf6a9dc994e997c63e1bfff/src/main/ua-parser.js#L96
